### DISCUSSION TITLE
Support for Compose Multiplatform

### DISCUSSION THREAD

**Discussion by rkreienbuehl** [sceneview/sceneview#492] [Ideas]
Are there any plans to support sceneview for Compose Multiplatform iOS?

**Comment by ThomasGorisse** [comment]
Not yet, sorry.
You will have to use [Filament](https://github.com/google/filament) directly for that.

**Comment by ThomasGorisse** [comment]
SceneView's cross-platform approach uses native renderers (Filament on Android, RealityKit on Apple) rather than Compose Multiplatform for rendering. This is because visionOS requires RealityKit, and each platform benefits from native tooling.

However, shared logic (math, collision, geometry, animations) is in `sceneview-core` — a KMP module. We also provide Flutter, React Native, and web (Kotlin/JS + Filament.js) bridges.

See #559 for more details. Latest: `io.github.sceneview:sceneview:4.0.0-rc.1`
